checklist.md

```diff
@@ -0,0 +1,32 @@
+# TodoApp Checklist
```

Comment:
perhaps, the checklist is not worth being here in the project
generally, I'd say it should mostly look like it's completely your project, your initiative